### ISSUE TITLE
Docker segment_name labels not work when scale containers

### ISSUE THREAD

**Issue by migueltercero** [containous/traefik#3482]
### Do you want to request a *feature* or report a *bug*?

bug

### What did you do?

We use docker backend to configure traefik with docker compose like this:
```yml
version: '3'
services:
  traefik:
    image: traefik
    command: 
      --web 
      --docker 
      --docker.domain=dev.local 
      --docker.watch 
      --docker.exposedbydefault=false
      --entryPoints='Name:http Address::80' 
      --defaultEntryPoints='http'
    ports:
      - "10000:80"
      - "10001:443"
      - "10002:8080"
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
  whoami:
    image: billsb/alpine-python-whoami
    labels:
      - "traefik.enable=true"
      - "traefik.frontend.rule=Host:mydomain.com"
      - "traefik.frontend.backend=whoami"
```
When we scale whoami service:

docker-compose up -d --scale whoami=2

traefik generate one backend with 2 containers like we expect:

![image](https://user-images.githubusercontent.com/20397404/41302728-87c50a30-6e6b-11e8-957b-e00a3f138b34.png)

But when we use segment_name (ours containers have multiple ports) like this:
```yml
version: '3'
services:
  traefik:
    image: traefik
    command: 
      --web 
      --docker 
      --docker.domain=dev.local 
      --docker.watch 
      --docker.exposedbydefault=false
      --entryPoints='Name:http Address::80' 
      --defaultEntryPoints='http'
    ports:
      - "10000:80"
      - "10001:443"
      - "10002:8080"
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
  whoami:
    image: billsb/alpine-python-whoami
    labels:
      - "traefik.enable=true"
      - "traefik.**service1**.frontend.rule=Host:mydomain.com"
      - "traefik.**service1**.backend=whoami"
```
traefik creates two backends with one container each and ignores traefik.<segment_name>.backend label:

![image](https://user-images.githubusercontent.com/20397404/41302844-dd0a43d4-6e6b-11e8-83ad-21d790899f44.png)

so traefik can´t balance request between instances.

### What did you expect to see?
![image](https://user-images.githubusercontent.com/20397404/41302889-f9077cfa-6e6b-11e8-98d1-af0cf1ba8815.png)

### What did you see instead?
![image](https://user-images.githubusercontent.com/20397404/41302897-ff78c4d6-6e6b-11e8-8d45-b98378100f69.png)

### Output of `traefik version`: (_What version of Traefik are you using?_)
```
Version:      v1.6.3
Codename:     tetedemoine
Go version:   go1.10.2
Built:        2018-06-05_03:29:01PM
OS/Arch:      linux/amd64
```

**Comment by ldez** [comment]
Duplicate of #3351

**Comment by ldez** [comment]
@migueltercero could you try to remove `"traefik.service1.frontend.backend=whoami"`

**Comment by ldez** [comment]
@migueltercero #3351  talk about `traefik.<segment_name>.frontend.backend`:


> Before 1.6.1:
> The label `traefik.backend` was partially ignored.
> The `traefik.<segment_name>.frontend.backend` label was used to name segment backend, this suggested that it was possible to set a backend manually for a frontend but it's not.
>
> Since 1.6.1:
> The label `traefik.backend` is fully used.
> The `traefik.<segment_name>.frontend.backend` label is removed for the benefit of `traefik.<segment_name>.backend`


The `traefik.<segment_name>.frontend.backend` label is removed for the benefit of `traefik.<segment_name>.backend`

**Comment by migueltercero** [comment]
Thanks for your quick response

Changed to:

    labels:
      - "traefik.enable=true"
      - "traefik.service1.frontend.rule=Host:mydomain.com"
      - "traefik.service1.backend=whoami"

scale whoami service

dc up -d --scale whoami=2 whoami

And result is the same, one frontend with two backends for the same service/segment_name

![image](https://user-images.githubusercontent.com/20397404/41335432-11786bdc-6ee9-11e8-94b4-65e75b116be6.png)

All request sent to **backend-tmp-whoami-1-whoami** backend.

Update conf over isssue to avoid confusion with #3351.

Thank you

**Comment by migueltercero** [comment]
This is the log when restart whoami:

traefik_1  | time="2018-06-13T07:39:08Z" level=debug msg="Configuration received from provider docker: {\"backends\":{\"backend-tmp-whoami-1-whoami\":{\"servers\":{\"server-service1-tmp-whoami-1-0\":{\"url\":\"http://172.24.0.4:8080\",\"weight\":1}},\"loadBalancer\":{\"method\":\"wrr\"}},\"backend-tmp-whoami-2-whoami\":{\"servers\":{\"server-service1-tmp-whoami-2-0\":{\"url\":\"http://172.24.0.3:8080\",\"weight\":1}},\"loadBalancer\":{\"method\":\"wrr\"}}},\"frontends\":{\"frontend-tmp-whoami-1-whoami\":{\"entryPoints\":[\"http\"],\"backend\":\"backend-tmp-whoami-1-whoami\",\"routes\":{\"route-frontend-tmp-whoami-1-whoami\":{\"rule\":\"Host:mydomain.com\"}},\"passHostHeader\":true,\"priority\":0,\"basicAuth\":[]}}}"

Pretty format:

{
	"backends": {
		"backend-tmp-whoami-1-whoami": {
			"servers": {
				"server-service1-tmp-whoami-1-0": {
					"url": "http://172.24.0.4:8080",
					"weight": 1
				}
			},
			"loadBalancer": {
				"method": "wrr"
			}
		},
		"backend-tmp-whoami-2-whoami": {
			"servers": {
				"server-service1-tmp-whoami-2-0": {
					"url": "http://172.24.0.3:8080",
					"weight": 1
				}
			},
			"loadBalancer": {
				"method": "wrr"
			}
		}
	},
	"frontends": {
		"frontend-tmp-whoami-1-whoami": {
			"entryPoints": ["http"],
			"backend": "backend-tmp-whoami-1-whoami",
			"routes": {
				"route-frontend-tmp-whoami-1-whoami": {
					"rule": "Host:mydomain.com"
				}
			},
			"passHostHeader": true,
			"priority": 0,
			"basicAuth": []
		}
	}
}

Like in web ui, one frontend --> two backends.

Regards.

**Comment by ldez** [comment]
I confirm the bug, it will be fixed in #3485

**Comment by traefiker** [comment]
Closed by #3485.
